Resources/Audio/Animals/license.txt

```diff
@@ -12,4 +12,27 @@ chicken_cluck_happy.ogg: modified from "Chicken Single Alarm Call" by freesound
 
 ferret_happy.ogg: modified from "Ferret" by freesound user "J.Zazvurek" (https://freesound.org/people/J.Zazvurek/sounds/155115/) licensed under CC BY 3.0. The original audio was trimmed and converted from WAV to OGG format.
 
-mouse_squeak.ogg: modified from "Cartoon - Bat / Mouse Squeak" by freesound user "Breviceps" (https://freesound.org/people/Breviceps/sounds/445958/) licensed under CC0 1.0 (public domain). The original audio was converted from WAV to OGG format.
\ No newline at end of file
+mouse_squeak.ogg: modified from "Cartoon - Bat / Mouse Squeak" by freesound user "Breviceps" (https://freesound.org/people/Breviceps/sounds/445958/) licensed under CC0 1.0 (public domain). The original audio was converted from WAV to OGG format.
+
+sloth_squeak.ogg: modified from "squeakfinal.wav" by freesound user "Higgs01" (https://freesound.org/people/Higgs01/sounds/428114/) licensed under CC0 1.0 (Public domain dedication). The original audio was converted from WAV to OFF format AND converted from stereo to mono.
+
+parrot_raught.ogg: modified and used from "Parrot 1 raught 2.wav" by freesound user "Coral_Island_Studios" (https://freesound.org/people/Coral_Island_Studios/sounds/432588/) licensed under CC BY 3.0. The original audio was converted from WAV to OGG format.
+
+fox_squeak.ogg: modified from "Video Game Squeak" by freesound user "Breviceps" (https://freesound.org/people/Breviceps/sounds/468442/) licensed under CC0 1.0. The original audio was converted from WAV to OGG format
+
+frog_ribbit.ogg: original audio created by freesound user "egomassive" (https://freesound.org/people/egomassive/sounds/536759/). licensed under licensed under CC0 1.0
+
+goose_honk.ogg: modified from " Bird Honk - 1"  by freesound user "SpaceJoe" (https://freesound.org/people/SpaceJoe/sounds/510940/) licensed under CC0 1.0.  The originial audio was converted from WAV to OGG format.
+
+snake_hiss.ogg: modified from "hissing snake sound effect" by freesound user "Garuda1982" (https://freesound.org/people/Garuda1982/sounds/541656/) licensed under CC BY 4.0. The original audio was convertred from WAV to OGG format.
+
+The following sounds were used from youtube:
+
+penguin_squawk.ogg: audio is recorded/provided by youtube user "ProSounds" (https://youtu.be/Anr35RbBL3Y) audio is free to use and downloadable via video description. The original audio was trimmed and converted from MP3 to OGG format.
+
+raccoon_squeak.ogg: audio is recorded by youtube user "jnargus" (https://youtu.be/BGjFP1CP7E0) licensed under Creative Commons Attribution license (reuse allowed)
+
+goat_bah.ogg: audio is provided by youtube user "Winry Marini" (https://youtu.be/QIhwzsk5bww) granted permission to be used in game
+
+lizard_happy.ogg: audio created by youtube user "Nagaty Studio" (https://youtu.be/I7CX0AS8RNI) licensed under Creative Commons Attribution license (reuse allowed).
+
```

Comment:
I'm not to sure about the sounds from youtube where you just state that we can use those sounds.
For the sounds that have a licence you should state the proper licence name like above.
In this case that would be: CC BY 3.0

Reply:
Alright I will do that. I did however just to let you know asked them directly just in case.